src/Form/ChangeDossierStatusType.php

```diff
@@ -27,7 +27,7 @@ class ChangeDossierStatusType extends AbstractType
     private $user;
     private $workflowRegistry;
 
-    public function __construct(TokenStorageInterface $tokenStorage, WorkflowRegistry $registry)
+    public function __construct(TokenStorageInterface $tokenStorage, ?WorkflowRegistry $registry)
```

Comment:
Wat doet de `?` hier precies?

Reply:
De ? geeft aan dat het type ofwel WorkflowRegistry mag zijn, ofwel dat er een null-object meegegeven mag worden. Er is namelijk tijdens het testen een situatie gezien waarin de workflowRegistry inderdaad null is, maar de applicatie niet blokkeert.